app/src/pages/api/upload.ts

```diff
@@ -31,7 +38,7 @@ export default async function handler(
     const form = formidable({});
     const [, files] = await form.parse(req);
 
-    const results = await Promise.allSettled(
+    const results: BlurryDetectorResults = await Promise.allSettled(
```

Comment:
Any idea what the requirements are around persisting files? Would we need these at some other point in the future for the user to reference? If so then [Vercel Blob storage](https://vercel.com/docs/storage/vercel-blob) or another 3rd party storage solution might be a good fit.